F4transkript/scripts/preinstall

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+process_list=(
+"f4transkript.app"
+)
+
+# Kill processes if any are given as argument.
+for process in "${process_list[@]}"; do
+  # Terminate gracefully
+  osascript -e "quit app \"$process\""
+  sleep 1
+  # Get process id and kill process. Exclude "PKInstallSandbox" to not accidentally terminate other installations.
+  for process_id in $(ps -ax | grep "$process" | grep -v "PKInstallSandbox" | grep -v "grep" | awk '{print $1}'); do
+    echo "INFO: Process '$process' with PID '$process_id' is running, will now kill forcefully!"
+    kill -9 "$process_id"
```

Comment:
This will meen a user loses all their unsafed data. I would prefer not to have a preinstall script and handle this in the munki pkgsinfo.plist.

Reply:
Danke, @aschwanb 
das werde ich dann über die pkgsinfo machen.